k8s-daemonset/k8s/hello-world-1_4.yml

```diff
@@ -66,26 +66,22 @@ spec:
       - name: service
         image: buoyantio/helloworld:0.0.1
         env:
-        - name: NODE_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: spec.nodeName
         - name: POD_IP
           valueFrom:
             fieldRef:
               fieldPath: status.podIP
         command:
         - "/bin/bash"
         - "-c"
-        - "HTTP_PROXY=$(NODE_NAME):4140 python world.py"
+        - "python world.py"
```

Comment:
can you talk about the changes to this file?  I think this will break the service mesh blog posts